loadtest/control/simulcontroller/controller.go

```diff
@@ -148,6 +148,12 @@ func getActionList(c *SimulController) []userAction {
 			frequency:        0.0040,
 			minServerVersion: control.MinSupportedVersion, // 5.26.0
 		},
+		{
+			name:             "UpdateCustomAttribute",
+			run:              c.updateCustomAttribute,
+			frequency:        0.0040,
+			minServerVersion: control.MinSupportedVersion, // 10.9.0
+		},
```

Comment:
We need to specify the version here:
```suggestion
		{
			name:             "UpdateCustomAttribute",
			run:              c.updateCustomAttribute,
			frequency:        0.0040,
			minServerVersion: semver.MustParse("10.9.0"),
		},
```